### DISCUSSION TITLE
Technical Preview - is it suitable for production use cases?

### DISCUSSION THREAD

**Discussion by philipahlberg** [neondatabase/neon#4874] [General]
Hi,

I'm considering using Neon for an existing application, and in that context, I have some questions.

- The term "technical preview" seems to suggest that it should not be used in a production environment. Is that correct?

- Is there a timeline for the technical preview, or a set of technical goals that need to be accomplished before Neon will exit technical preview?

**Comment by hab25** [comment]
I'm also interested in both of those answers.
Related:  
https://neon.tech/docs/introduction/roadmap
https://vercel.com/storage/postgres (beta)
